charts/super-agent-deployment/templates/rbac.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.rbac.create -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    {{- include "newrelic.common.labels" . | nindent 4 }}
+  name: {{ include "newrelic.common.naming.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups:
+      - source.toolkit.fluxcd.io
+      - helm.toolkit.fluxcd.io
+      - kustomize.toolkit.fluxcd.io
```

Comment:
Kustomize is disabled for now, we could remove this as well I think, unless you think it could be useful for some workarounds

Reply:
Done in [#2e7ea0a](https://github.com/newrelic/helm-charts/pull/1155/commits/2e7ea0a72b79ef91f8fdc94f8231798b34553883)